### DISCUSSION TITLE
the Dynamic  proxy

### DISCUSSION THREAD

**Discussion by Globalfun32y439y54** [UseInterstellar/Interstellar#676] [General]
Where did you guys find the Dynamic proxy?

**Comment by replitidk** [comment]
go to settings click on proxys and then click dynamic proxy you cant use this proxy in the about blank but it will let you search also if you are using github code spaces then you wont be able to use GeForce now if you want to use GeForce now then you need a interstellar link or use gitpod @Globalfun32y439y54

**Reply by nvd123YT** [reply]
He's asking HOW they found it not what it does

**Comment by nvd123YT** [comment]
Dynamic is trash ngl
